docs: update stackblitz code version by fixing name of <live-example>

ShubhankarKG opened the pull request:
The previous life example had the name "...-v0". This caused the products.ts in the tutorial to not have an id field.
Later on, during navigation, readers had to deduce that they need to add an id field manually.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)


## PR Type
What kind of change does this PR introduce?

<!-- Please check the one that applies to this PR using "x". -->

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:


## What is the current behavior?
<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->

Issue Number: #40280 


## What is the new behavior?

The `<live-example>` points to the new `products.ts` file that has `id` for each product.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No


<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. -->

ShubhankarKG committed:
docs: update stackblitz code version by fixing name of <live-example>

The previous life example had the name "...-v0". This caused the products.ts in the tutorial to not have an id field.
Later on, during navigation, readers had to deduce that they need to add an id field manually.

ShubhankarKG commented:
@googlebot I signed it!

mary-poppins commented:
You can preview c883e08 at https://pr40284-c883e08.ngbuilds.io/.

petebacondarwin reviewed (changes requested):
I don't think it is quite right to change this link to point to the "final" application, since then there is nothing for people to do as they are following along with the tutorial.

Instead I think we should update the `v0` example to contain what is needed at: https://github.com/angular/angular/tree/master/aio/content/examples/getting-started-v0

ShubhankarKG committed:
docs: update v0 products with id

The getting-started-v0 had products.ts that did not have id fields.
This can cause an issue in the navigation part. This commit adds an id field to all products.

ShubhankarKG commented:
Sorry about that, did not realize I had linked the final application. :sweat_smile:

mary-poppins commented:
You can preview 3ed4567 at https://pr40284-3ed4567.ngbuilds.io/.

aikithoughts reviewed (approved):
I'll approve, pending @petebacondarwin provides final approval here.

Reviewed-for: global-docs-approvers

aikidave commented:
Is this change overlapping with PR 40197?

ShubhankarKG commented:
Yes, looks like it. This change is a subset of #40197.

petebacondarwin commented:
@ShubhankarKG - thanks for working on this PR but since it is a subset of #40197, which is about to be merged, I am going to go ahead and close this PR.

ShubhankarKG commented:
Sure
